package.json

```diff
@@ -24,6 +24,7 @@
     "@typescript-eslint/eslint-plugin": "^4.15.1",
     "@typescript-eslint/parser": "^4.15.1",
     "babel-eslint": "^10.1.0",
+    "bad-words-next": "^1.6.7",
```

Comment:
Why in root? It's not mono repo tool or something global. Move dep to condo :)

Reply:
You right. Fixed.